### PR TITLE
Fix backbone session expiration

### PR DESCRIPTION
**Pull request by jistr**
This renames the `check_session` before filter to `check_session_expiration` and fixes a low-severity bug that I found previously [1]. Hopefully it also makes the filter code a bit more straightforward.

Tested with specs/cukes on F17 + rubygems and F16 + rpm gems. (Some specs fail on F17 + rubygems, as usual.) Also tested manually on F17.

[1] https://github.com/aeolusproject/conductor/pull/19#issuecomment-8094708

### CODE REVIEW

**Comment by jguiditta** [comment]
Could you please add a spec/requests regression test to describe this scenario?

File: src/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -460,23 +460,18 @@ def paginate_collection(collection, page = nil, per_page = PER_PAGE)
...
+
     # FIXME: we really need a better "is it backbone?" test than json format
     # since other actions (ajax API calls from UI, etc) could also use json
-    if request.format == :json
-      # compare to last non-backbone reqest time
-      last_time = session[:last_active_request]
-      if last_time
-        logout if last_time < SETTINGS_CONFIG[:session][:timeout].minutes.ago
-      else
-        # This isn't really an active reqest, but
```
Comment (jguiditta) [inline src/app/controllers/application_controller.rb:473]: Super minor note, s/reqest/request above

**Comment by jistr** [comment]
Spec added, typo fixed :)

The spec fails with master and passes with my pull request.

**Comment by jistr** [comment]
Revision 2 of this pull request is at https://github.com/aeolusproject/conductor/pull/43

**Pull request closed without merging**